packages/payments-plugin/e2e/paypal-sdk.plugin.ts

```diff
@@ -0,0 +1,136 @@
+import { Controller, Get, Header, Inject, Type } from '@nestjs/common';
+import { Ctx, RequestContext } from '@vendure/core';
+import { PluginCommonModule, VendurePlugin } from '@vendure/core';
+import { DocumentNode, print } from 'graphql';
+
+import { ADD_PAYMENT_TO_ORDER, CREATE_PAYPAL_ORDER } from './payment-helpers';
+
+const PAYPAL_SDK_PAYMENT_PLUGIN_OPTIONS = Symbol('PAYPAL_SDK_PAYMENT_PLUGIN_OPTIONS');
+export interface PayPalSdkPluginOptions {
+    apiUrl: string;
+    clientId: string;
+}
+
+// @ts-ignore
+function html(strings, ...values) {
```

Comment:
Well. I know we shouldn't integrate code only for syntax highlighting, but as this isn't code of the library itself and only exists for testing purposes, one could argue that it's still okay. 
Especially as it doesn't require additional dependencies.